### DISCUSSION TITLE
Need to remove a PR for a undisclosed security advisory

### DISCUSSION THREAD

**Discussion by tigerinus** [community/community#60206] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

One of our team members submitted a PR publicly for a critical security advisory instead of fixing it in a private fork. The PR is closed but the code change can still be seen. Deleting the branch doesn't help.

Please suggest what's our best move next. Thanks!

**Comment by LucaScorpion** [comment, marked as answer]
I believe you need to file a ticket with support. From here: https://support.github.com/request/remove-data you can see a button "Remove pull requests", I think that should be just what you need.

**Comment by GhostOf0days** [comment]
I've used this [approach](https://blog.avneesh.tech/how-to-delete-all-commit-history-in-github) before. Be aware it will clean your commit history except for latest commit.

**Reply by GhostOf0days** [reply]
You could also use [this](https://support.github.com/request/remove-data) as the other person who replied stated.
